src/metadata.rs

```diff
@@ -0,0 +1,92 @@
+use crate::LiveFile;
+use crate::{ffi, ffi_util::from_cstr};
+use std::ffi::CString;
+use std::fmt::Debug;
+use std::ptr;
+
+/// Metadata for column family import/export.
+#[derive(Debug)]
+pub struct ExportImportFilesMetaData {
+    pub(crate) inner: *mut ffi::rocksdb_export_import_files_metadata_t,
+}
```

Comment:
Would be nice if this support Clone.

Also, Send.

Reply:
Ack, I saw why Send is useful when I used this in Restate yesterday :-)

Reply:
I'm going to hold off on Clone for now - that would require me to rebuild the `Vec<LiveFile>` by calling the FFI layer again, and I don't think it's _that_ necessary in practice. I think most users will typically be re-hydrating this struct from some serialized form.